### PR TITLE
Updated stanford_media

### PR DESCRIPTION
**Pull request by pookmish**
Relies on https://github.com/SU-SWS/stanford_media/pull/24

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "drupal/editor_advanced_link": "^1.4",
         "drupal/embed": "~1.0",
         "drupal/linkit": "~5.0@beta",
-        "su-sws/stanford_media": "dev-8.x-2.x"
+        "su-sws/stanford_media": "^8.2.0"
```
Comment (sherakama) [inline composer.json]:
```suggestion
        "su-sws/stanford_media": "^8.2.0@alpha"
```

Comment (pookmish) [reply composer.json]: they do the same.

Comment (sherakama) [reply composer.json]: Sort of but this allows for alpha builds when the `minimum-stability` is set to stable instead of dev.

Comment (pookmish) [reply composer.json]: cool

**Pull request merged**